src/clr-angular/utils/drag-and-drop/interfaces/drag-event.ts

```diff
@@ -3,15 +3,23 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-
 export enum ClrDragEventType {
     DRAG_START,
     DRAG_MOVE,
     DRAG_END,
     DROP
 }
 
+export interface ClrDragDataTransfer {
+    // TODO: groupLinks seems to be a good name?
+    groupLinks?: string|string[];
+    data: any;
```

Comment:
Why declare a sub-interface for this? It seems a bit overkill, having the group and the payload at the root of the event seems easier, especially when you're going to have to update them independently sometimes.

Reply:
Addressed the request.